promql/lex.go

```diff
@@ -198,21 +198,20 @@ var key = map[string]itemType{
 	"stdvar": itemStdvar,
 
 	// Keywords.
-	"alert":         itemAlert,
-	"if":            itemIf,
-	"for":           itemFor,
-	"labels":        itemLabels,
-	"annotations":   itemAnnotations,
-	"offset":        itemOffset,
-	"by":            itemBy,
-	"without":       itemWithout,
-	"keeping_extra": itemKeepCommon,
-	"keep_common":   itemKeepCommon,
-	"on":            itemOn,
-	"ignoring":      itemIgnoring,
-	"group_left":    itemGroupLeft,
-	"group_right":   itemGroupRight,
-	"bool":          itemBool,
```

Comment:
Apologies, there's no "reverse map", I meant the list of tokens in https://github.com/prometheus/prometheus/blob/master/promql/lex.go#L110-L183